components/relay/Cargo.toml

```diff
@@ -10,10 +10,18 @@ serde_json = "1"
 thiserror = "1.0"
 url = "2"
 viaduct = { path = "../viaduct" }
+uniffi = "0.29.0"
 
 [dev-dependencies]
 env_logger = { version = "0.10", default-features = false, features = ["humantime"] }
 expect-test = "1.4"
 mockall = "0.11"
 mockito = "0.31"
 viaduct-reqwest = { path = "../support/viaduct-reqwest" }
+
+[build-dependencies]
+uniffi = { version = "0.29.0", features=["build"]}
+
+[lib]
+name = "relay"
+crate-type = ["cdylib", "staticlib"]
```

Comment:
You probably also want `rlib` here if you're going to list the crate types.  That way the example can still use it.

Stepping back, how are you going to distribute this component?  In general, we bundle all our components into one of the megazord libraries and ship those to FF android/ios/desktop.  If you're doing the same thing, you can leave `crate-type` out, since the megazord's crate type is what matters in those cases.

Reply:
I'll remove this `[lib]` section completely - most other components don't include it in their `Cargo.toml`.